.github/workflow/pylint.yml

```
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example:
+    runs-on: ubuntu-latest
+    name: Executes pylint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Lint
+        uses: gabriel-milan/action-pylint@v1
+        with:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}} # GitHub token
+          path: "./*.py" # Glob pattern for files to lint
```

Not super familiar with GitHub action globbing, but I would expect it to be this pattern based on the documentation https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

```suggestion
          path: "**.py" # Glob pattern for files to lint
```

---

> Not super familiar with GitHub action globbing, but I would expect it to be this pattern based on the documentation https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Nice catch. Thanks. Fixed.